episodes/03-create.md

```diff
@@ -105,45 +236,76 @@ $ git status
 ```output
 On branch main
 
-No commits yet
-
-nothing to commit (create/copy files and use "git add" to track)
+nothing to commit, working tree clean
 ```
 
 If you are using a different version of `git`, the exact
 wording of the output might be slightly different.
 
+::::::::::::::::: checklist
+
+### Checklist
+
+![Use the `git init` command to initialize a Local Repository in your Workspace. Use `git status` to check the status of the repository.](fig/cut-git-verb_map-02.png)
+
+:::::::::::::::::::::::::::
+
+The steps done with `{usethis}` can also be done with commands in the Terminal. For example, instead of `usethis::use_git()` in the Console you can use `git init` in the Terminal. However, we prefer using the first one given their explicit messages, interactivity, and warnings to prevent errors!
+
+::::::::::::::::: spoiler
+
+### How to ask help to Git?
```

Comment:
Would "How to ask Git for help?" be a more clear wording?

Reply:
I included this edit in https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/b685ede9026723788a3b9db58c7e8f64747cead6